packages/camera/camera/README.md

```diff
@@ -17,9 +17,7 @@ A Flutter plugin for iOS, Android and Web allowing access to the device cameras.
 * Record video.
 * Add access to the image stream from Dart.
 
-## Installation
-
-First, add `camera` as a [dependency in your pubspec.yaml file](https://flutter.dev/using-packages/).
```

Comment:
I'm curious as to why this is being deleted

Reply:
I dropped this in response to Stuart's suggestion in https://github.com/flutter/packages/pull/6963#discussion_r1648273732 that this section isn't super useful with pub.dev surfacing installation information. Perhaps I misunderstood how much I should remove though.

Reply:
Sgtm